### ISSUE TITLE
Still Sprockets 3 issue

### ISSUE THREAD

**Issue by ai** [Vasfed/csso-rails#24]
```
NameError: uninitialized constant Sprockets::Compressors
/home/ai/Dev/amplifr/.bundle/gems/csso-rails-0.4.0/lib/csso.rb:16:in `install'
/home/ai/Dev/amplifr/.bundle/gems/csso-rails-0.4.0/lib/csso/railtie.rb:7:in `block in <class:Railtie>'
```

/cc @Vasfed

**Comment by kuwabarahiroshi** [comment]
+1

**Comment by mada41** [comment]
+1 , i still get same issue

**Comment by phortx** [comment]
+1

**Comment by chrise86** [comment]
+1

**Comment by Vasfed** [comment]
Should be fixed in 0.4.1
